src/copy.js

```diff
@@ -7,8 +7,30 @@ chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
   }
 });
 
+/**
+ * Regex to match twitter and x.com URLs.
+ */
+const FIX_TARGET = /twitter\.com|x\.com/g
+
+/**
+ * Regex to match tweet URLs.
+ */
+const TWITTER_URL_REGEX = /https?:\/\/twitter\.com\/[a-zA-Z0-9_]+\/status\/[0-9]+/g;
+
 document.addEventListener('copy', e => {
-  currentLink = document.location.href;
+  /**
+   * If window.location.href is a tweet link, put it in the clipboard.
+   * If not, check if there is a selected text and put it in the clipboard.
+   */
+  currentLink = window.location.href.match(TWITTER_URL_REGEX) ? window.location.href : window.getSelection().toString();
```

Comment:
Countermeasures for copying with ⌘+k. If it's a link from a tweet, just copy it.

Reply:
This should work in many cases, but it is incomplete. If there is a better way, I would like you to think about it with me.